packages/browser/src/index.ts

```diff
@@ -157,42 +159,28 @@ export default class LogtoClient {
       return accessToken.token;
     }
 
-    // Token expired, remove it from the map
-    if (accessToken) {
-      this.accessTokenMap.delete(accessTokenKey);
-    }
+    /**
+     * Token has expired, need to fetch a new one using refresh token.
+     * Reuse the cached promise if exists.
+     */
+    const cachedPromise = this.getAccessTokenPromiseMap.get(accessTokenKey);
```

Comment:
I think we should remove the accessToken from `accessTokenMap` in here, rather than doing it in function `getAccessTokenByRefreshToken` in line 301.

Reply:
this will cause duplicate removals if we call the function multiple times

Reply:
get it.

Reply:
> this will cause duplicate removals if we call the function multiple times

Seems not. See #232